propolis/src/hw/pci/device.rs

```diff
@@ -53,13 +64,8 @@ impl<D: Device + Send + Sync + 'static> Endpoint for D {
     }
     fn cfg_rw(&self, mut rwo: RWOp, ctx: &DispCtx) {
         let ds = self.device_state();
-        ds.cfg_space.process(&mut rwo, |id, mut rwo| match id {
-            CfgReg::Std => {
-                STD_CFG_MAP.process(&mut rwo, |id, rwo| match rwo {
-                    RWOp::Read(ro) => ds.cfg_std_read(id, ro, ctx),
-                    RWOp::Write(wo) => ds.cfg_std_write(self, id, wo, ctx),
-                });
-            }
+        ds.cfg_space.process(&mut rwo, |id, rwo| match id {
+            CfgReg::Std => self.std_cfg_rw(rwo, ctx),
```

Comment:
I'm not entirely certain about the necessity of this rearranging.  I imagined that a more fully fleshed out bridge device might impl `Endpoint` directly, rather than `Device`, since it'd be doing some things differently around its BARs and interrupts.